app/modules/passthroughs/resources.py

```diff
@@ -155,14 +97,14 @@ def post(self, target, path):
         except Exception:
             pass
 
-        request_func = current_app.edm.post_passthrough
-
         passthrough_kwargs = {'data': data}
 
         files = request.files
         if len(files) > 0:
             passthrough_kwargs['files'] = files
 
-        response = _request_passthrough(target, path, request_func, passthrough_kwargs)
+        response = current_app.edm.request_passthrough(
+            'passthrough.data', 'get', passthrough_kwargs, path, target
```

Comment:
Is this supposed to use `'post'` argument instead of `'get'` since it is in the `post` method?